src/store/index.js

```diff
@@ -7,13 +7,13 @@ export const DerivStore = React.createContext()
 
 export const DerivProvider = ({ children }) => {
     const [website_status] = useWebsiteStatus()
-    const [is_eu_country, setEuCountry] = useState(false)
+    const [is_eu_country, setEuCountry] = useState(null)
     const [is_p2p_allowed_country, setP2PAllowedCountry] = useState(false)
     const [crypto_config, setCryptoConfig] = useState(null)
 
     useEffect(() => {
         if (website_status) {
-            setEuCountry(isEuCountry(website_status.clients_country))
+            setEuCountry(!!isEuCountry(website_status.clients_country))
```

Comment:
just out of curiosity...if the usestate is set to false instead of null, this !! can be omitted right?

Reply:
It's actually setting !! based on `isEuCountry` and not `is_eu_country`.